### ISSUE TITLE
Spam 

### ISSUE THREAD

**Issue by PopuleuxMusicYT** [InfiniteFlightAirportEditing/Airports#4874]
# Airport Name

Bates Field

# Country?

America

# Improvements that need to be made?

Airport doesn't exist, must be made from scratch

# Are you working on this airport?

yes, i am starting it

# Airport Priority? (IF Event, 10000ft+ Runway, World/US Capital, Low)

Low Priority

**Comment by MajorTomMueller** [comment]
You are assigned to 3 issues already. Do not open further issues before a previous one passed OR. 

P.S
When I am still in bad moods by coming home I will redo this myself...
